stable/datadog/templates/daemonset.yaml

```diff
@@ -127,6 +127,14 @@ spec:
             path: {{ .Values.datadog.containerLogsPath | quote }}
           name: logcontainerpath
         {{- end }}
+        {{- if .Values.datadog.customLogFilesEnabled }}
+        {{- range $_, $value := .Values.datadog.customLogFiles }}
```

Comment:
It can be simplified to `{{- range .Values.datadog.customLogFiles }}` as you did in `container-agent.yaml`. Consequently instead of using `$value.<field>` let's just use `.<field>`.